layouts/blog-index.hbs

```diff
@@ -7,16 +7,30 @@
 
     <main>
         <div class="container">
-
-            <ul>
-                {{#each collections.blog}}
-                    {{#if title}}
-                        <li>
-                            <a href="/{{../../site.locale}}/{{ path }}">{{ title }}</a>
-                        </li>
-                    {{/if}}
-                {{/each}}
-            </ul>
+            
+            <article>
+                {{#with collections.blog.[0]}}
+                    <h3 class="blog-title"><a href="/{{../site.locale}}/{{path}}">{{title}}</a></h3>
+                    <div class="blog-blurb">
+                        {{{ contents }}}
+                    </div>
+                    <p class="blog-ellipses">...</p>
+                    <a href="/{{../site.locale}}/{{path}}">Read the full post</a>
```

Comment:
Well, this was just posting a blurb of the most recent post. I added the blurb because the entries were long. I can change it to be the full post in, or leave the blurb.